Makefile

```diff
@@ -27,5 +27,6 @@ test_mw: phpunit_mw
 
 phpunit_mw:
 	php .mediawiki/tests/phpunit/phpunit.php -c .mediawiki/vendor/wikibase/mediawiki-term-store/phpunit.xml.dist ./tests/Unit/MediaWikiDependent
+	php .mediawiki/tests/phpunit/phpunit.php -c .mediawiki/vendor/wikibase/mediawiki-term-store/phpunit.xml.dist ./tests/Integration
```

Comment:
I'd move this into it's own task. say `test_integration` and chain it in `ci` task instead of adding it here because integration tests will not always be mw dependent ones.